py-polars/polars/internals/type_aliases.py

```diff
@@ -71,6 +71,7 @@
 RollingInterpolationMethod: TypeAlias = Literal[
     "nearest", "higher", "lower", "midpoint", "linear"
 ]  # QuantileInterpolOptions
+CorrelationMethod: TypeAlias = Literal["pearson", "spearman"]  # CorrelationOptions
```

Comment:
This should be in the block below, under things that do not have an equivalent on the Rust side.

Reply:
@stinodego , I moved `CorrelationMethod` to the code block below.